taroscript/send.go

```diff
@@ -313,28 +312,24 @@ func SignVirtualTransaction(input *taropsbt.VInput, outputs []*taropsbt.VOutput,
 	// input of the virtual TX, generate a witness, and attach it to the
 	// copy of the new Asset.
 	//
-	// TODO(guggero): I think this is wrong... We shouldn't look at
-	// PrevWitnesses of the single asset we spend but instead have multiple
-	// inputs if we want to spend multiple coins of the same asset?
-	prevWitnessCount := len(newAsset.PrevWitnesses)
-	for idx := 0; idx < prevWitnessCount; idx++ {
-		prevAssetID := newAsset.PrevWitnesses[idx].PrevID
-		prevAsset := prevAssets[*prevAssetID]
-		virtualTxCopy := VirtualTxWithInput(
-			virtualTx, prevAsset, uint32(idx), nil,
-		)
-
-		newWitness, err := SignTaprootKeySpend(
-			*input.Asset().ScriptKey.RawKey.PubKey, virtualTxCopy,
-			prevAsset, 0, txscript.SigHashDefault, signer,
-		)
-		if err != nil {
-			return err
-		}
+	// TODO(guggero): Update this once we want to support more than one
+	// asset being spent in a single transaction.
```

Comment:
See other comments re leaving the existing logic as it should work once we allow multiple inputs. We can also do simpler exercises of this by doing things like send two collectibles via an interactive flow in itests.